files/collector/otel-collector-config.yml

```diff
@@ -0,0 +1,31 @@
+receivers:
+  otlp:
+    protocols:
+      grpc:
+  zipkin:
+
+exporters:
+  zipkin:
+    endpoint: "http://zipkin:9411/api/v2/spans"
++  newrelic:
++    apikey: 'ADD_A_LICENSE_KEY_HERE'
++    timeout: 30s
++    spans_host_override: "trace-api.newrelic.com"
```

Comment:
Should these `+`'s be here?

Reply:
Appears to be a copy from a git diff.  I've done this one too many times too.

Reply:
I think this is just from a `git diff` - agreed they should be removed.

Viewing the raw file makes things slightly more clear:
https://github.com/open-telemetry/opentelemetry-php/blob/d24984cfb72e4c02016fef3637842948344a2b84/files/collector/otel-collector-config.yml#L10-L13

Reply:
:) thanks for catching this!!!!